src/types/device-authentication/index.ts

```diff
@@ -31,16 +33,14 @@ import { Plugin, Cordova, IonicNativePlugin } from '@ionic-native/core';
 })
 @Injectable()
 export class DeviceAuthentication extends IonicNativePlugin {
-
   /**
    * Run device authentication prompt. If FaceId/TouchId is set it shows prompt to use either one of them. If none of them are set it show passcode screen.
    * @param message {string} Message that is show in authentication prompt
    * @return {Promise<boolean>} Returns a promise that resolves when user tried to authenticate
    * @throws {Promise<string>} Returns a promise that rejects when there are some errors. Returned string contains localized message
    */
   @Cordova()
-  runAuthentication(message: string): Promise<boolean> {
+  runAuthentication(_message: string): Promise<boolean> {
```

Comment:
Again, why the underscore?

Reply:
Hi Mathieu, its connected with https://github.com/dvsa/mobile-examiner-alpha/pull/55#discussion_r180983592 (can we continue thread there?)